include/wlr/types/wlr_tablet_v2.h

```diff
@@ -0,0 +1,130 @@
+#ifndef WLR_TYPES_WLR_TABLET_V2_H
```

Comment:
We don't have a consistent naming scheme for unstable protocol implementations.
1. Some implementations are missing the version number, we should fix that.
2. We're stripping the "z" prefix. That means that if a xdg-shell stable v2 protocol is publised, it will conflict with unstable versions.